src/components/views/messages/MAudioBody.js

```diff
@@ -34,13 +34,15 @@ export default class MAudioBody extends React.Component {
             error: null,
         };
     }
+
     onPlayToggle() {
         this.setState({
             playing: !this.state.playing,
         });
     }
 
     _getContentUrl() {
+        if (this.props.forExport) return "forExport";
```

Comment:
Actually, this is a little rarer than this since we replace (src/href)="forExport" (including quotes). But yes, using mxcs makes more sense. Modified this.